app/assets/javascripts/automate_import_export.js

```diff
@@ -213,6 +214,14 @@ var Automate = {
   },
 
   setUpGitRefreshClickHandlers: function() {
+    var toggleSubmitButton = function() {
+      if ($('.git-branch-or-tag').val() !== "") {
```

Comment:
This should probably be `$('.git-import-submit').prop('disabled', $('.git-branch-or-tag').val() !== "");